Automattic/wp-calypso
# Add difm design picker step

#### Changes proposed in this Pull Request

The current DIFM lite flow, design selection is extremely painful to the customer. But since typeform is limited in customizability this change bring that step out of the flow into a design picker with preview available a well. The design is also more mobile friendly for design selection.

| Mobile      |  Desktop  |
| ----------- | ----------- |
| <img src="https://user-images.githubusercontent.com/3422709/135597429-82f26e87-e974-4ef3-a8aa-0660e10c7d72.png" alt="drawing" width="200"/>| ![difm-design-picker2](https://user-images.githubusercontent.com/3422709/135596197-7f3a90b4-38b8-4e25-9739-b95ed3f66e85.gif)         |





#### Testing instructions

- Go to `/start/do-it-for-me`
- Check whether design picker works as shown above
